api/v1alpha1/kubevirtcluster_types.go

```diff
@@ -43,10 +43,16 @@ type KubevirtClusterSpec struct {
 	ControlPlaneEndpoint APIEndpoint `json:"controlPlaneEndpoint,omitempty"`
 
 	// SSHKeys is a reference to a local struct for SSH keys persistence.
+	// +optional
 	SshKeys SSHKeys `json:"sshKeys,omitempty"`
 
 	// InfraClusterSecretRef is a reference to a secret with a kubeconfig for external cluster used for infra.
+	// +optional
 	InfraClusterSecretRef *corev1.ObjectReference `json:"infraClusterSecretRef,omitempty"`
+
+	// LoadBalancerServiceTemplate describes load balancer service template.
+	// +optional
+	LoadBalancerServiceTemplate LoadBalancerServiceTemplate `json:"loadBalancerServiceTemplate,omitempty"`
```

Comment:
It may make sense to explicitly state what exactly gets routed over this service. Like: apiserver traffic, ingress, ...

maybe it makes sense to not call this `LodaBalancer...` since in fact we can also choose non-load-balancer service types and instead use a name which hints what exactly is routed over this service.

Reply:
agree, will update. historically, we called it a "load balancer" in a sense of it being a load balancer for control plane nodes, even though as far as implementation goes it's a simple service. maybe make more sense to name it "control plane service" in the spec to be generic, and add more details in the api docs as you suggest 👍